dashboard/settings.py

```diff
@@ -44,15 +45,21 @@
 GA_ID = ENV.get('GA_ID', '')
 
 # Resource values from env
-RESOURCE_VALUES = ENV.get("RESOURCE_VALUES", {"files": {"types": ["canvas"], "icon": "fas fa-file fa-lg"}})
+RESOURCE_VALUES: Dict[str, Any] = ENV.get("RESOURCE_VALUES", {"files": {"types": ["canvas"], "icon": "fas fa-file fa-lg"}})
 
 # Convience map to be able to get from types
-RESOURCE_VALUES_MAP = {
+RESOURCE_VALUES_MAP: Dict[str, str] = {
     resource_type : resource_value
-    for resource_value in RESOURCE_VALUES
-    for resource_type in RESOURCE_VALUES.get(resource_value).get('types')
+    for resource_key, resource_value in RESOURCE_VALUES.items()
+    for resource_type in resource_value.get('types')
 }
 
+def get_resource_icon(resource_type: str) -> str:
+    resource_value = RESOURCE_VALUES.get(str(RESOURCE_VALUES_MAP.get(resource_type)))
+    if resource_value:
+        return resource_value.get('icon')
+    return ''
+
```

Comment:
This seems like a good addition, but I'd probably put it in `dashboard/common/utils`.